### PR TITLE
{data} [intel-2014b] MySQLdb 1.2.5 Python 2.7.8 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: easybuild/easyconfigs/m/MySQLdb/MySQLdb-1.2.5-intel-2014b-Python-2.7.8.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+easyblock = "PythonPackage"
+
+name = 'MySQLdb'
+version = '1.2.5'
+
+homepage = [
+    'http://mysql-python.sourceforge.net/',
+    'https://pypi.python.org/pypi/MySQL-python/',
```
Comment (boegel) [inline easybuild/easyconfigs/m/MySQLdb/MySQLdb-1.2.5-intel-2014b-Python-2.7.8.eb]: move up the PyPI source URL first (or only, even), sourceforge is considered evil these days...

Comment (boegel) [inline easybuild/easyconfigs/m/MySQLdb/MySQLdb-1.2.5-intel-2014b-Python-2.7.8.eb]: ah, wait, this is homepage... just use the PyPI URL here (homepage should be a string, not a list)

**Comment by boegel** [comment]
@hajgato: lgtm, but why the old toolchain & Python?

**Comment by hajgato** [comment]
@boegel The last three request come form the same user, so this is why intel-2014b. Anyway, this was not a big deal.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/51cbc69fc0a43f5bc477 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**